### ISSUE TITLE
[ChatQnA Conversational UI] require root privilege for the UI

### ISSUE THREAD

**Issue by Ruoyu-y** [opea-project/GenAIExamples#517]
Implementation of the Conversational UI requires root privilege to run. It brings potential risk to the system when the UI is running as a pod within Kubernetes such as privilege escalation, container escape, etc.

**Comment by eero-t** [comment]
> Implementation of the Conversational UI requires root privilege to run

Why?  Node / JS in general does not require that...

**Comment by xiguiw** [comment]
@Ruoyu-y  
Is this issue resolved?

**Comment by xiguiw** [comment]
@jaswanth8888 

It decreases the security of OPEA. This is a critical issue. 

Pleased resolve is ASAP.

**Comment by eero-t** [comment]
I don't think this is specific to ChatQnA, I think it concerns all UIs containers?

@xiguiw please tag also related ticket in "GenAIInfra": https://github.com/opea-project/GenAIInfra/issues/500

**Comment by lianhao** [comment]
I've updated the bug description to be more precise. It's a common security bug for all svelte and react UIs.

**Comment by eero-t** [comment]
Ok, first issue seems to be wrong read access to config file, and second one is temp files being written to wrong location.  If you make those 2 locations writable by non-root, are there additional issues?

**Comment by sgurunat** [comment]
@xiguiw - Will look into this issue for React UI

**Comment by xiguiw** [comment]
> I don't think this is specific to ChatQnA, I think it concerns all UIs containers?
> 
> [@xiguiw](https://github.com/xiguiw) please tag also related ticket in "GenAIInfra": [opea-project/GenAIInfra#500](https://github.com/opea-project/GenAIInfra/issues/500)

@eero-t  
Yes.  [opea-project/GenAIInfra#500 UI pod is run with full privileges](https://github.com/opea-project/GenAIInfra/issues/500) has dependency on this.

**Comment by xiguiw** [comment]
[Remind] 
@WenjiaoYue Please look into this issue for svelte UI

**Comment by yinghu5** [comment]
Just update for remind.
